extension/src/popup/components/mnemonicPhrase/MnemonicDisplay/index.tsx

```diff
@@ -1,8 +1,50 @@
 import React from "react";
 import { Card } from "@stellar/design-system";
+// @ts-ignore
+import { generateMnemonic } from "stellar-hd-wallet";
+import { random } from "lodash";
 
 import "./styles.scss";
 
+interface generateMnemonicPhraseDisplayProps {
+  mnemonicPhrase: string;
+}
+
```

Comment:
can we add a comment here explaining the reason for the random words? For future readers who may not have context

Reply:
Good thinking!